samples/core/lightweight_component/lightweight_component.ipynb

```diff
@@ -2,26 +2,28 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "source": [
     "# Lightweight python components\n",
     "\n",
     "Lightweight python components do not require you to build a new container image for every code change.\n",
     "They're intended to use for fast iteration in notebook environment.\n",
     "\n",
     "#### Building a lightweight python component\n",
-    "To build a component just define a stand-alone python function and then call kfp.components.func_to_container_op(func) to convert it to a component that can be used in a pipeline.\n",
+    "To build a component just define a stand-alone python function with the `@dsl.component` decorator to convert it to a component that can be used in a pipeline.\n",
     "\n",
     "There are several requirements for the function:\n",
     "* The function should be stand-alone. It should not use any code declared outside of the function definition. Any imports should be added inside the main function. Any helper functions should also be defined inside the main function.\n",
     "* The function can only import packages that are available in the base image. If you need to import a package that's not available you can try to find a container image that already includes the required packages. (As a workaround you can use the module subprocess to run pip install for the required package. There is an example below in my_divmod function.)\n",
     "* If the function operates on numbers, the parameters need to have type hints. Supported types are ```[int, float, bool]```. Everything else is passed as string.\n",
-    "* To build a component with multiple output values, use the typing.NamedTuple type hint syntax: ```NamedTuple('MyFunctionOutputs', [('output_name_1', type), ('output_name_2', float)])```"
+    "* To build a component with multiple output values, use the typing.NamedTuple type hint syntax: ```NamedTuple('MyFunctionOutputs', output_name_1=type), output_name_2=type, ...)```. You can access a named output using ```.output['<output-key>']```."
```

Comment:
`.output['<output-key>']` should be `.outputs['<output-key>']` (plural) instead.

Reply:
Thanks, that makes sense. I do agree.  
Note that the public documents[1][2], we described as singular form and I took the notation as it is. 

Let me fix the documentation as well. 

[1] https://www.kubeflow.org/docs/components/pipelines/v2/pipelines/pipeline-basics/#data-passing-and-task-dependencies
[2] https://www.kubeflow.org/docs/components/pipelines/v2/data-types/parameters/#multiple-output-parameters

Reply:
Done